### ISSUE TITLE
Feature Request: flag to print multiple labels?

### ISSUE THREAD

**Issue by sailhobie** [din14970/elabftwqrprint#3]
might be nice to have a flag that prints multiple labels for when producing multiple samples from experiment.

could do a for loop at bash level I suppose?

**Comment by din14970** [comment]
Might indeed be nice to have a function that retrieves a list of ID's via a kind of fuzzy search or so, which can be piped to a print function. I'll see what a good way to implement it would be, but this one may take a bit longer.

**Comment by din14970** [comment]
Now in v0.0.5 you should be able to use the `print_sticker_elab_item` with multiple sample ID's and they should print sequentially. Give it a try to see if it works for you. They will all be passed the same optional arguments though, so if you specify long text it will be the same on all stickers.

I've also improved search functionality and you can pipe the results of the search into the print command, e.g.. 

```
find_items -s "copper" -m 20200301 | print_sticker_elab_item
```

**Comment by sailhobie** [comment]
Awesome! I'll try to take a look tonight after the kids go to bed.

**Comment by sailhobie** [comment]
Hey, 
When I print  multiple items it prints the number of items but the label is just a replicate.
for example,
`print_sticker_elab_item 3 4`
results in two labels for #3

BTW the label orientation for 12mm continuous labels is working great

**Comment by din14970** [comment]
Yes I overlooked a small thing - the stickers look the same but the QR codes should be unique and work for the ones you put in. It's only the short description below that was not updating. I fixed this issue and uploaded v0.0.6.

**Comment by din14970** [comment]
I now tested it on my printer and it seems to work. If you experience any other issues or feature requests let me know, I will close off this one.
